packages/contracts-core/contracts/test/Replica.t.sol

```diff
@@ -453,9 +453,9 @@ contract ReplicaTest is ReplicaHandlers {
 
     function test_setOptimisticTimeoutOnlyOwner() public {
```

Comment:
Ideally, we should revisit this and fuzz it, but good choice for something more realistic.